protos/bottle/fulfillment/v1/line_group.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineGroup {
+  string id = 1;
+  string name = 2;
+  int32 quantity = 3;
+
+  bottle.catalog.v1.Product catalog_product = 4;
+
+  enum LineGroupCategory {
+    LINE_GROUP_CATEGORY_UNSPECIFIED = 0;
+    LINE_GROUP_CATEGORY_DESKTOP = 1;
+    LINE_GROUP_CATEGORY_LAPTOP = 2;
+    LINE_GROUP_CATEGORY_SERVER = 3;
+    LINE_GROUP_CATEGORY_APPAREL = 4;
+  }
+  LineGroupCategory category = 5;
+
+  bool rush = 6;
+  double rush_price = 7;
+
+  enum LineGroupWarrantyType {
+    LINE_GROUP_WARRANTY_TYPE_UNSPECIFIED = 0;
+    LINE_GROUP_WARRANTY_TYPE_STANDARD = 1;
+    LINE_GROUP_WARRANTY_TYPE_EXTENDED = 2;
+  }
+  LineGroupWarrantyType warranty_type = 8;
+  double warranty_price = 9;
+
+  double individual_cost = 10;
+  double individual_price = 11;
```

Comment:
And I labeled these `individual_` because they don't take into account the `quantity` from above.